editor.planx.uk/src/@planx/components/Send/Public.tsx

```diff
@@ -52,6 +55,20 @@ const SkipSendWarning: React.FC<Props> = (props) => (
   </Card>
 );
 
+const DemoTeamWarning: React.FC<Props> = (props) => (
+  <Card handleSubmit={props.handleSubmit}>
+    <ErrorSummaryContainer role="status">
+      <Typography variant="h4" ml={2} mb={1}>
+        Send is disabled for the Demo team
```

Comment:
Similar to above! Is "Demo team" public-facing language we want to use? Also let's pick either "disabled" or "not enabled" for consistency!